experimental/packages/propagator-aws-xray-lambda/package.json

```
@@ -0,0 +1,69 @@
+{
+  "name": "@opentelemetry/propagator-aws-xray-lambda",
+  "version": "1.22.0",
```

This is intended to be experimental, right?

```suggestion
  "version": "0.49.1",
```